R/tune_results.R

```diff
@@ -66,6 +66,7 @@ summarize_notes <- function(x) {
     dplyr::group_nest(type) %>%
     dplyr::mutate(data = purrr::map(data, ~ dplyr::count(.x, note))) %>%
     tidyr::unnest(data) %>%
+    dplyr::rowwise() %>%
```

Comment:
The changes in this file fix a bug in `summarize_notes()` where the condition messages for warnings and errors were pasted if there were multiple distinct notes. I indulged myself and went ahead and made this fix so that we could snapshot test that case well, but committed those changes separately so that we could document what that test output looks like without those changes.